src/main/kotlin/com/openlattice/organizations/DatasetController.kt

```diff
@@ -160,8 +160,8 @@ class DatasetController : DatasetApi, AuthorizingComponent {
             @PathVariable(ID) organizationId: UUID,
             @PathVariable(TABLE_NAME) tableName: String,
             @RequestBody metadataUpdate: MetadataUpdate) {
-        ensureAdminAccess()
         val tableFqnToId = getExternalDatabaseObjectFqnToIdPair(organizationId, tableName)
+        ensureOwnerAccess(AclKey(tableFqnToId.second))
```

Comment:
should that be `ensureWriteAccess`?

Reply:
I'm not entirely sure how Write/Read/Owner map to actual postgres permissions actually but I agree that Admin is too much and that it should be either Owner or Write